public/data/LukeMurray.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Luke Murray",
+    "type": "personal",
+    "bio": "Technical Consultant, Azure Solutions Architect Expert, Technologist and a drinker of coffee.",
+    "avatar": "https://luke.geek.nz/images/ProfileRealLife.jpg",
+    "links": [
+      {
+        "name": "Follow me on my Website",
+        "url": "https://luke.geek.nz",
+        "icon": "globe"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/lukemurraynz",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/lukemurraynz",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/ljmurray",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Follow me on Microsoft Tech Community",
+        "url": "https://techcommunity.microsoft.com/t5/user/viewprofilepage/user-id/108279#profile",
+        "icon": "microsoft"
+      }   
+    
+    ]
+  }
```

Comment:
```suggestion
{
  "name": "Luke Murray",
  "type": "personal",
  "bio": "Technical Consultant, Azure Solutions Architect Expert, Technologist and a drinker of coffee.",
  "avatar": "https://luke.geek.nz/images/ProfileRealLife.jpg",
  "links": [
    {
      "name": "Follow me on my Website",
      "url": "https://luke.geek.nz",
      "icon": "globe"
    },
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/lukemurraynz",
      "icon": "twitter"
    },
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/lukemurraynz",
      "icon": "github"
    },
    {
      "name": "Follow me on LinkedIn",
      "url": "https://www.linkedin.com/in/ljmurray",
      "icon": "linkedin"
    },
    {
      "name": "Follow me on Microsoft Tech Community",
      "url": "https://techcommunity.microsoft.com/t5/user/viewprofilepage/user-id/108279#profile",
      "icon": "microsoft"
    }     
  ]
}
```

Reply:
Please do not make suggestions related to formatting since it is now done by the GH action 🙂

Reply:
Ohh, great 🙌